tests/test_gpu_openmp.py

```diff
@@ -257,4 +257,4 @@ def test_iso_ac(self):
 
         op(time=time_range.num-1, dt=dt)
 
-        assert np.isclose(norm(rec), 490.55, atol=1e-2, rtol=0)
+        assert np.isclose(norm(rec), 491.55, atol=1e-2, rtol=0)
```

Comment:
this is scary

Reply:
Hah, I guess I should have put a comment above. This was changed to enforce a fail to ensure fails are being reported correctly :p